packages/x-components/src/x-modules/ai/store/getters/suggestions-request.getter.ts

```diff
@@ -12,8 +12,8 @@ import type { AiXStoreModule } from '../types'
  * @public
  */
 export const suggestionsRequest: AiXStoreModule['getters']['suggestionsRequest'] = (
-  { params },
+  { params, origin },
   { query },
 ) => {
-  return { query, lang: params.lang as string, extraParams: params }
+  return { query, extraParams: params, origin: origin ?? undefined }
```

Comment:
I would leave the origin as it is in the state as the rest of modules. Value or null